src/services/annotations-parser.js

```diff
@@ -0,0 +1,155 @@
+import { getCanvasId } from "./iiif-parser";
+import { getLabelValue, getMediaFragment, parseTimeStrings } from "./utility-helpers";
+
+export function parseAnnotationSets(manifest, canvasIndex) {
+  let canvas = null;
+  let annotationSets = [];
+
+  // return empty object when canvasIndex is undefined
+  if (canvasIndex === undefined || canvasIndex < 0) {
+    return null;
+  }
+
+  // return an error when the given Manifest doesn't have any Canvas(es)
```

Comment:
I'm not seeing an error getting returned for this case (looks like it is `{canvasIndex, []}`).
Should this comment be removed or the error added?

Reply:
Thanks for catching this! The comment should be removed. I have a test case handling this use-case in the other [PR](https://github.com/samvera-labs/ramp/pull/753).

Reply:
@cjcolvar In the other [PR](https://github.com/samvera-labs/ramp/pull/753), it returns `null` instead of `{ canvasIndex, annotationSets: [] }`. I can make this change in there if the latter makes more sense.